web/src/features/hfiCalculator/slices/hfiCalculatorSlice.ts

```diff
@@ -57,21 +58,22 @@ export interface HFICalculatorState {
   numPrepDays: number
   selectedPrepDate: string
   startDate: string
-  planningAreaFireStarts: { [key: string]: FireStarts[] }
+  planningAreaFireStarts: { [key: string]: FireStartRange[] }
   planningAreaHFIResults: { [key: string]: PlanningAreaResult }
   selectedFireCentre: FireCentre | undefined
   result: HFIResultResponse | undefined
   saved: boolean
 }
 
 export interface HFIResultResponse {
+  // TODO: Change from string to SmartDate
```

Comment:
I presume this TODO is for a subsequent PR?

Reply:
Yep - again depending on the feedback I get on smartdate.